tools/openms/IDFilter.xml

```diff
@@ -70,6 +70,17 @@
 #if $param_whitelist_by_seq_only:
   -whitelist:by_seq_only
 #end if
+
+#if $rep_param_whitelist_protein_accessions:
+-whitelist:protein_accessions
+  #for token in $rep_param_whitelist_protein_accessions:
+    #if " " in str(token):
+      "$token.param_whitelist_protein_accessions"
```

Comment:
you mean 
"$token.param_whitelist_protein_accessions"
and not:
$token.param_whitelist_protein_accessions

ever?